assets/json-schema-faker.js

```diff
@@ -24302,7 +24299,7 @@ function extend() {
   
     // properties are read from right-to-left
     const _limit = optionalsProbability !== null || requiredProperties.length === max ? max : random.number(0, max);
-    const _props = requiredProperties.concat(random.shuffle(extraProperties).slice(0, _limit)).slice(0, max);
+    const _props = requiredProperties.concat(extraProperties).slice(0, max);
```

Comment:
@umeshp7 New updated JSF code was causing some issue. i.e. here properties were shuffled so they were in not same order as in schema. So removed logic of shuffling.

PR that changed this: https://github.com/postmanlabs/openapi-to-postman/pull/450/files